content/blog/tidymodels-sparse-support/index.Rmd

```diff
@@ -0,0 +1,146 @@
+---
+output: hugodown::hugo_document
+slug: tidymodels-sparse-support
+title: Sparse data structures in tidymodels
+date: 2020-11-16
+author: Julia Silge
+description: >
+    Sparse data is common in many domains, and now tidymodels supports using 
+    sparse matrix structures throughout the fitting and tuning stages of modeling.
+photo:
+  url: https://unsplash.com/photos/7JX0-bfiuxQ
+  author: JJ Ying
+# one of: "deep-dive", "learn", "package", "programming", or "other"
+categories: [learn] 
+tags: [tidymodels,tune,parsnip,hardhat]
+
+---
+
+
+```{r setup, include=FALSE}
+library(knitr)
+knitr::opts_chunk$set(cache = TRUE, cache.lazy = FALSE, warning = FALSE, 
+                      message = FALSE, echo = TRUE, dpi = 180,
+                      fig.width = 8, fig.height = 5)
+library(tidyverse)
+theme_set(theme_minimal())
+```
+
+The new release of [tune](linktopost) is chock full of improvements and new features. When combined with the latest releases of [hardhat](http://hardhat.tidymodels.org/) and [parsnip](https://parsnip.tidymodels.org/), one upgrade that tidymodels users can now use in their day-to-day modeling work is **support for sparse data structures**.
```

Comment:
Mention that this is the second of three articles related to tune 0.1.2 and use link to previous (which should be `/blog/2020/11/tune-0-1-2/`)